libs/books/data-access/src/lib/+state/reading-list.actions.ts

```diff
@@ -32,6 +33,21 @@ export const removeFromReadingList = createAction(
   props<{ item: ReadingListItem }>()
 );
 
+export const markedAsFinished = createAction(
```

Comment:
I would change the tense here, since it indicates an intention. So, I would call it `markAsFinished`. Same goes for the action text, `[Reading List] Mark as finished`.